src/Xlang/UndockedRegFreeWinRT/src/UndockedRegFreeWinRT/Catch/catch.hpp

```diff
@@ -7394,7 +7394,7 @@ namespace Catch {
                 result = -erfc_inv(2.0 * p);
                 // result *= normal distribution standard deviation (1.0) * sqrt(2)
                 result *= /*sd * */ ROOT_TWO;
-                // result += normal disttribution mean (0)
+                // result += normal distribution mean (0)
```

Comment:
It's possible this is treated as an upstream file and should be in excludes... (I need to see if I have corrected the upstream...)

Reply:
Another one for the engineering team to review.